packages/@aws-cdk/aws-servicecatalogappregistry/test/integ.application.ts

```diff
@@ -9,5 +9,4 @@ new appreg.Application(stack, 'TestApplication', {
   description: 'my application description',
 });
 
-app.synth();
-
+app.synth();
```

Comment:
Why this change? I think CDK files should end with a newline.

Reply:
added back